bin/oref0-pump-loop.sh

```diff
@@ -240,6 +240,10 @@ function smb_bolus {
     # and administer the supermicrobolus
     find enact/ -mmin -5 | grep smb-suggested.json > /dev/null \
     && if (grep -q '"units":' enact/smb-suggested.json); then
+        # press ESC three times on the pump to exit Bolus Wizard before SMBing, to help prevent A52 errors
+        openaps use pump press_keys esc | jq .completed | tr '\n' ' '
```

Comment:
`openaps use pump press_keys esc esc esc` works for me, returning an array of three confirmations.  It would lead to less process start overhead to use that form.

Reply:
Thanks.  And that also gives a single summary boolean for `completed`, which we can use to bail out and not SMB if false.